pumpkin-plugin/Cargo.toml

```diff
@@ -5,4 +5,5 @@ edition.workspace = true
 
 [dependencies]
 extism = "1.5.0"
-log.workspace = true
\ No newline at end of file
+log.workspace = true
+serde ={ version = "1.0.209", features = ["derive"] }
```

Comment:
Hey can you may just use serde version `1.0`. I really should move serde into workspace

Reply:
Sure!

Reply:
I can move it into the workspace if you're fine with it